src/tree/updater_quantile_hist.h

```diff
@@ -218,13 +235,41 @@ class QuantileHistMaker: public TreeUpdater {
     const DMatrix* p_last_fmat_;
 
     using ExpandQueue =
-        std::priority_queue<ExpandEntry, std::vector<ExpandEntry>,
-                            std::function<bool(ExpandEntry, ExpandEntry)>>;
-    std::unique_ptr<ExpandQueue> qexpand_;
+       std::priority_queue<ExpandEntry, std::vector<ExpandEntry>,
+                           std::function<bool(ExpandEntry, ExpandEntry)>>;
+
+    std::unique_ptr<ExpandQueue> qexpand_loss_guided;
```

Comment:
I think we should suffix all class variables with `_` to differentiate them from local variables. Example: `qexpand_loss_guided_`.